app/poros/json_schema_validator.rb

```diff
@@ -42,10 +42,11 @@ def schema_validation_errors
   rescue *[
     Errno::ENOENT,
     JSON::Schema::JsonParseError,
+    JSON::Schema::ReadFailed,
     JSON::Schema::SchemaParseError,
   ]
     if Rails.env.test?
-      puts("absolute_schema_path: #{absolute_schema_path}")
+      puts("absolute_schema_path: #{absolute_schema_path rescue nil}")
```

Comment:
I will be surprised if this is necessary, but I'm adding it just in case possibly somehow `absolute_schema_path` does raise an error.